frontend/src/pages/AccountSend.vue

```diff
@@ -248,12 +256,19 @@ function useSendForm() {
         await humanAmountInputRef.validate();
       }
 
+      const currentTokens = (tokenListValue as TokenInfoExtended[]).map((tok) => tok.address);
+      const prevTokens = (prevTokenListValue as TokenInfoExtended[]).map((tok) => tok.address);
+      if (currentTokens.toString() != prevTokens.toString()) {
+        token.value = tokenList.value[0];
+        humanAmount.value = undefined;
+      }
```

Comment:
I had needed this to avoid some noisy errors, but when I just removed it now I wasn't able to reproduce them. Not sure what happened exactly. I'll leave it out